frontend/components/TableContainer/DataTable/TruncatedTextCell/TruncatedTextCell.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState, useRef, useLayoutEffect } from "react";
+import { uniqueId } from "lodash";
+
+import ReactTooltip from "react-tooltip";
+
+interface ITruncatedTextCellProps {
+  value: string | number | boolean;
+  classes?: string;
+}
+
+const TruncatedTextCell = ({
```

Comment:
This component was super finicky to create with the truncated css. I didn't want to mix it in with the TextCell css. Because we have so many different Cell types, I decided to make a new cell.